playbooks/common/openshift-cluster/reset_excluder.yml

```diff
@@ -5,4 +5,12 @@
   tasks:
   - include_role:
       name: openshift_excluder
+      tasks_from: init
+  - include_role:
+      name: openshift_excluder
       tasks_from: reset
```

Comment:
The tricky part is going to be persisting the data about whether the host should be re-enabled or not...  For that, it might be good to take advantage of a local fact or persisting the status to a local file on the control host, to avoid issues with re-running the upgrade on a failed upgrade and leaving a node out of rotation.